lighthouse-core/test/results/sample_v2.json

```diff
@@ -3233,7 +3233,44 @@
             }
           }
         ],
-        "summary": {}
+        "summary": {},
+        "debugData": {
+          "type": "debugdata",
+          "vulnerableLibs": [
+            {
+              "name": "jQuery",
+              "npmPkgName": "jquery",
+              "version": "2.1.1",
+              "vulns": [
+                {
+                  "severity": "Medium",
+                  "numericSeverity": 2,
+                  "library": "jQuery@2.1.1",
+                  "url": "https://snyk.io/vuln/SNYK-JS-JQUERY-567880"
+                },
+                {
+                  "severity": "Medium",
+                  "numericSeverity": 2,
+                  "library": "jQuery@2.1.1",
+                  "url": "https://snyk.io/vuln/SNYK-JS-JQUERY-565129"
+                },
+                {
+                  "severity": "Medium",
+                  "numericSeverity": 2,
+                  "library": "jQuery@2.1.1",
+                  "url": "https://snyk.io/vuln/SNYK-JS-JQUERY-174006"
+                },
+                {
+                  "severity": "Medium",
+                  "numericSeverity": 2,
+                  "library": "jQuery@2.1.1",
+                  "url": "https://snyk.io/vuln/npm:jquery:20150627"
+                }
+              ],
+              "highestSeverity": "Medium"
+            }
```

Comment:
Even filtered this still seems mostly redundant. I guess you get to see the different severities per vulnerability, but the majority in our snyk db are `medium` anyways. Do we want just `['https://snyk.io/vuln/SNYK-JS-JQUERY-567880', 'https://snyk.io/vuln/SNYK-JS-JQUERY-565129', 'https://snyk.io/vuln/SNYK-JS-JQUERY-174006', 'https://snyk.io/vuln/npm:jquery:20150627']` out of this, which captures almost all of the unique additional data? The link we currently provide (https://snyk.io/vuln/npm:jquery?lh=2.1.1) does already have all that for interested parties...

Reply:
@paulirish Thoughts?

Reply:
I think there are two cases for use here:

a human or script looking at an individual LHR
  - most of the time you just want to know what you're using that's vulnerable and that you should upgrade
  - if you want to dig in, everything you'd want is already at the URL we provide for each library found (that page links off to all the vulnerabilities individually, severities, CVEs, etc)
  - if a tool really want to show all that detail in their own UI, snyk has both an API and local databases for library@version -> vulnerabilities :)

otoh, if one were to do an HTTP Archive query about vulnerable libraries, I'd imagine the things you'd want out of the LHR are, in decreasing order of likelihood of interest:
- the vulnerable version of the library found in a page
- the highest vulnerability level found (e.g. select results only from the pages with a High severity vulnerability)
- the number of vulnerabilities found for that library

I can't imagine any circumstance in which you'd want the set of all vulnerability levels found for that library (`['Medium', 'Medium', 'Medium', 'Medium']`), for instance. What would you do with that?

I think this is another case of YAGNI. Regardless of possible scenarios, in practice no one has missed this extra info since `extendedInfo` went away, so let's add it back if and when someone has a real need for it. That also nicely avoids having to come up with which properties to keep and which to ditch and what shape object should they be put in, etc :)

Reply:
sounds good. let's drop.